app/src/forms/email/assets/confirmation-number-email.html

```diff
@@ -1,118 +1,259 @@
-          <p class="MsoNormal" style="margin-bottom:0cm; margin-bottom:.0001pt; line-height:normal">
-            &nbsp;</p>
-        </td>
-      </tr>
+                                              </div>
+                                            </td>
+                                          </tr>
+                                          <tr>
+                                            <td data-slot-container="1" align="center" style="
+                                            font-size: 0px;
+                                            padding: 10px 25px;
+                                            word-break: break-word;
+                                          ">
+                                              <div data-slot="text" style="
+                                              color: #777777;
+                                              font-family: Oxygen, Helvetica neue,
+                                                sans-serif;
+                                              font-size: 14px;
+                                              line-height: 21px;
+                                              text-align: left;
+                                            ">
+                                                <p>
+                                                  <span style="font-size:12.0pt; color:#595959">{{ messageLinkText
+                                                    }}</span>
+                                                </p>
+                                                <p>
+                                                  <a href="{{ messageLinkUrl }}" style="
+                                                    background-color: #003366;
+                                                    color: #ffffff;
+                                                    border-radius: 6px;
+                                                    font-family: Oxygen, Helvetica neue, sans-serif;
+                                                    font-size: 16px;
+                                                    font-weight: 700;
+                                                    line-height: 37px;
+                                                    text-decoration: none;
+                                                    padding: 10px 15px;
+                                                  ">
+                                                    View Submission
+                                                  </a>
+                                                </p>
+                                              </div>
+                                            </td>
+                                          </tr>
+                                        </table>
+                                      </td>
+                                    </tr>
+
+                                </table>
+                            </td>
+                          </tr>
+
+
+
+
```

Comment:
Large number of unnecessary whitespace lines. Please make sure to run the VSCode formatter in the right click menu to make sure things get compacted and indented correctly.